blocks/init/src/Blocks/components/layout-three-columns/layout-three-columns.php

```diff
@@ -55,26 +56,26 @@
 
 ?>
 
-<<?php echo esc_attr($htmlTag); ?> class="<?php echo \esc_attr($layoutClass); ?>">
+<<?php echo esc_attr($layoutThreeColumnsHtmlTag); ?> class="<?php echo \esc_attr($layoutClass); ?>">
 	<div class="<?php echo \esc_attr($wrapClass); ?>">
 
-		<?php if ($layoutLeft) { ?>
+		<?php if ($layoutThreeColumnsLeft) { ?>
 			<div class="<?php echo \esc_attr($columnLeftClass); ?>">
-				<?php echo wp_kses_post(Components::ensureString($layoutLeft)); ?>
+				<?php echo wp_kses_post(Components::ensureString($layoutThreeColumnsLeft)); ?>
 			</div>
 		<?php } ?>
 
-		<?php if ($layoutCenter) { ?>
+		<?php if ($layoutThreeColumnsCenter) { ?>
 			<div class="<?php echo \esc_attr($columnCenterClass); ?>">
-				<?php echo wp_kses_post(Components::ensureString($layoutCenter)); ?>
+				<?php echo wp_kses_post(Components::ensureString($layoutThreeColumnsCenter)); ?>
 			</div>
 		<?php } ?>
 
-		<?php if ($layoutRight) { ?>
+		<?php if ($layoutThreeColumnsRight) { ?>
 			<div class="<?php echo \esc_attr($columnRightClass); ?>">
-				<?php echo wp_kses_post(Components::ensureString($layoutRight)); ?>
+				<?php echo wp_kses_post(Components::ensureString($layoutThreeColumnsRight)); ?>
```

Comment:
No need to kses here (since we're rendering other components here which should kses themselves)

Reply:
I was thinking the same, but the `ensureString` method is by itself not safe 🤷🏼‍♂️

Reply:
I mean it's the same thing as with other "wrapper / layout" blocks. It's not safe by itself, but it should be safe because of the fact that other simple / complex blocks escape their own content.

Ensure string is not safe but it does absolutely nothing other than concatenate multiple component outputs. It allows you to pass 1 or more components inside any column by sending a string (Component render output) or an array of strings (array of Component outputs).

Tho you could in theory still pass unsafe component in it, but then that's on you. 😄